examples/Backtesting/botorch_analytical.py

```diff
@@ -83,34 +84,21 @@
     mode="SINGLE", targets=[NumericalTarget(name="Target", mode="MIN")]
 )
 
-### Constructing campaigns for the simulation loop
-
-# To simplify adjusting the example for other strategies, we construct some strategy objects.
-# For details on strategy objects, we refer to [`strategies`](./../Basics/strategies.md).
-
-seq_greedy_EI_strategy = TwoPhaseStrategy(
-    recommender=SequentialGreedyRecommender(acquisition_function_cls="qEI"),
-)
-random_strategy = TwoPhaseStrategy(recommender=RandomRecommender())
-
-# We now create one campaign per strategy.
+### Constructing campaigns
 
 seq_greedy_EI_campaign = Campaign(
     searchspace=searchspace,
-    strategy=seq_greedy_EI_strategy,
     objective=objective,
 )
 random_campaign = Campaign(
     searchspace=searchspace,
-    strategy=random_strategy,
+    strategy=TwoPhaseStrategy(recommender=RandomRecommender()),
```

Comment:
you should be able to now just write `strategy=RandomRecommender()`

Reply:
Was able to do so, hence resolved.